tests/macos/xcodeproj/Single-Application-Project-AllTargets.xcodeproj/xcshareddata/xcschemes/Single-Application-RunnableTestSuite.xcscheme

```diff
@@ -27,18 +27,8 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       onlyGenerateCoverageForSpecifiedTargets = "NO"
-      shouldUseLaunchSchemeArgsEnv = "NO">
+      shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
-         <TestableReference
```

Comment:
why is the testable reference gone?

Reply:
it's back now, turns out having targets under test action is what triggers XCodeGen side to include the TestableReference Section (what tests to run)